src/mixer/samplerbank.cpp

```diff
@@ -10,8 +11,17 @@
 #include "track/track.h"
 #include "util/assert.h"
 
-SamplerBank::SamplerBank(PlayerManager* pPlayerManager)
+namespace {
+
+const ConfigKey kConfigkeyLastImportExportDirectory(
+        "[Samplers]", "LastImportExportDirectory");
```

Comment:
For new COs we wanted to use snake_case, I think 2e should use that for new config keys as well.

Reply:
okay. what's "2e"?

Reply:
ahhh "**we** should use that for new config keys as well

All done, ready again.